src/test/resources/alma-fix/990177418660206441.json

```diff
@@ -109,7 +109,7 @@
     "label" : "Mystik & Aszese des 16.-19. Jahrhunderts [Nationallizenz]",
     "type" : [ "Collection" ]
   }, {
-    "id" : "https://lobid.org/collections#NLZ",
+    "id" : "http://lobid.org/collections#NLZ",
     "type" : [ "Collection" ],
```

Comment:
If possible we should stick to `https`. Even more so if this was so before. Or do you have an argument that support your deliberately switching this?

Reply:
I was remembering: https://github.com/hbz/lobid-resources/issues/1343#issuecomment-1153739524
That we had to switch back to http instead of https.
And thought this was an error on our side.

Then I checked:

[aleph.lobid.org/resources/TT050372336.json](https://aleph.lobid.org/resources/TT050372336.json)
It seems to have used https for the collection URIs.

I roll back this change then.